Makes/ViewController.h

```diff
@@ -0,0 +1,14 @@
+//
+//  ViewController.h
+//  Makes
+//
+//  Created by Mohamed Ragab on 1/8/17.
+//  Copyright © 2017 Mohamed Ragab. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface ViewController : UIViewController <UICollectionViewDataSource,UICollectionViewDelegateFlowLayout>
+@property (nonatomic, strong) UICollectionView *makesCollectionView;
```

Comment:
As you don't need this property to be public one (you will not use it in other classes) so you should create it in the implementation file